pinot-query-planner/src/test/resources/queries/GroupByPlans.json

```diff
@@ -13,6 +13,20 @@
           "\n"
         ]
       },
+      {
```

Comment:
Can you also cover that JOIN related test case here where I commented about this hint being a NO-OP ? Just want to see the plan with and without hint